### PR TITLE
Change language as defined by NF Legal

### PR DESCRIPTION
**Pull request by bnb**
Original PR contents: 
>This change has been approved at the Board level but was simplifed from changes we'd proposed months ago in THIS PR. Due to the leaving of the executive director and a misunderstanding in how we have historically processed charter changes from the CommComm or the TSC to the Board, this took some time to weed out language vs. changes for things such as CC --> CommComm.
>
>This provides the minimal language changed as defined by Node Foundation Legal in order to make the CommComm Director codified in the charter but as mentioned, was originally provided by CommComm to the Board as similar language.
>
>**it has been suggested for future PRs that we make sure to tease out important governance changes from copy changes for clarity's sake.

Original PR author: @hackygolucky 

Original PR link (reverted for further discussion): https://github.com/nodejs/community-committee/pull/380

### CODE REVIEW

**Comment by refack** [comment]
FTR: I don't have a proposed change. I just wanted to make sure we have all the information, and that the changes that do land are those that have been approved.

What I did notice is the the current change (addition of 7.5) is self conflicting, and seemingly repetitive

7.4
> The CommComm will elect from amongst voting CommComm members a CommComm Chairperson to work on building an agenda for CommComm meetings and collaborate with the Individual Membership Directors the wishes of the CommComm to the Board for a term of one year according to the Node.js Foundation’s By-laws. The CommComm shall hold annual elections to select a CommComm Chairperson; ___there are no limits on the number of terms a CommComm Chairperson may serve.___

7.5
> The CommComm will elect from amongst voting CommComm members: 
> * a CommComm Chairperson, responsible for preparing agendas and coordinating CommComm meetings, 
> * and 
a representative to serve on the Node.js Foundation Board of Directors pursuant to its Bylaws. 
> * ___Each person shall serve a one year term, and may serve for more than one term. The same person may not hold both positions at the same time.___

I wanted to make sure the final approved language was to add paragraph (5) or to replace paragraph (4).

**Comment by MylesBorins** [comment]
@refack what was approved by the board was this exact language. Keep in mind that the "true charter" is an amalgamation of all the motions approved by the board. We could in theory go through all those motions and redraft a charter and have that approved. So it is important to think about the "intent" or "spirit" of the charter. That does not mean we should not fix this, more that it isn't a huge deal to have it temporarily not make as much sense.

I've opened https://github.com/nodejs/community-committee/pull/385 to simplify the text, this is based on the current text found in the TSC charter. Notice that it does remove the bit regarding the individual member director /cc @williamkapke

**Comment by refack** [comment]
I've been viewing https://youtu.be/r1WY6v00NhM?t=708, and AFAICT there was no Board resolution, rather they asked us to create the PR, and send it back to the Board for final approval. Also its still not clear to me if the wording is an addition or a replacement.

Also there's a typo in this PR:
> image

WRT the wording presented to the board:

> image

i.e. last line should stand on it own.

It's really hard to tweeze out board resolutions from youtube videos, especially since `Exhibit 1` and `Exhibit 2` are not available. 
@MylesBorins @hackygolucky @williamkapke @nodejs/board Where can I find the minutes of that meeting (2018-09-05)?

**Comment by refack** [comment]
To make myself clear. Since this change was authored and coordinated by @hackygolucky I would like their review and approval.

**Comment by MylesBorins** [comment]
@refack the original post by @bnb assigned @hackygolucky as a reviewer on this

**Comment by hackygolucky** [comment]
I'm not sure what we want here as I was the original PR post, so I don't think this makes sense for me to +1? The point of process is to get two +1s from CommComm members, mark as DO NOT MERGE, and then the Board approves what we've already approved. Once that occurs. It is ready for merge. I can merge, I shouldn't +1 my own PR.

**Review by amiller-gh — APPROVED** [review]
Because of the merge mixup, I'm okay with +1-ing this PR as-is as a way to help smooth over the process with legal. I see no ambiguity in that language here that should in any way prevent it from landing.

After it lands, we can fast-follow with the clarifying updates in #385. 

If I understand correctly – if we can reach consensus on both this PR and the follow up changes in #385 – we can bring both to the next board meeting. This PR can land first, and we can land #385 once the board and legal are done with their sign-off.

Yay ✨PROCESS ✨!

@nodejs/community-committee, lets get our +1s on this PR to help expedite things and then all dig in to craft #385 into something we're proud of.

**Comment by bnb** [comment]
I can't technically add a +1 since I created this PR, but consider this my +1.

**Review by mhdawson — APPROVED** [review]
LGTM

**Review by WaleedAshraf — APPROVED** [review]

File: Community-Committee-Charter.md
Comment (williamkapke) [inline Community-Committee-Charter.md]: Not two anymore

File: Community-Committee-Charter.md
Comment (JemBijoux) [inline Community-Committee-Charter.md]: nit: can we get some spaces around these bullets? Unless the content following on the next line is meant to be part of this bullet point...

**Review by jemjam — APPROVED** [review]

**Review by keywordnew — APPROVED** [review]

**Review by codeekage — APPROVED** [review]

File: Community-Committee-Charter.md
Comment (amiller-gh) [reply Community-Committee-Charter.md]: Lets make this change on #385 👍

**Comment by amiller-gh** [comment]
This has been open for a while with 7/15 giving explicit approval, and no explicit opposition – we're going to have Tracy bring this to the board this week for approval and then we can merge it in  🎉  

Please pile on to #385 to make further language changes. Given our velocity the past week, I don't think we'll have that PR ready to present by this week's board meeting, but I'd like to prioritize it so we can bring it to the next one for approval. Go forth and govern 😉

**Review by thefourtheye — APPROVED** [review]

**Comment by joesepi** [comment]
LGTM

**Review by hackygolucky — APPROVED** [review]

**Pull request merged**
